contracts/AddressResolver.sol

```diff
@@ -16,6 +16,11 @@ contract AddressResolver is Owned {
 
         for (uint i = 0; i < names.length; i++) {
             repository[names[i]] = destinations[i];
+            (bool success, ) =
+                address(destinations[i]).call(
+                    abi.encodePacked(MixinResolver(0).invalidateCache.selector)
+                );
+            success; // supressing the compiler warning
```

Comment:
Since a migration could potentially remap resolved addresses in contracts that implement the MixinResolver, this tells all the proxies that will be updated, that their address mappings are out of date.